src/App.tsx

```diff
@@ -1,54 +1,123 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
 import './App.scss';
+import cn from 'classnames';
+import debounce from 'lodash.debounce';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+  const [query, setQuery] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+  const [visible, setVisible] = useState(false);
```

Comment:
Use more understandable variables names. `visible` does not provide sense for what this variable.

Reply:
Correct. Fix, please.